components/table/TimesheetTable.tsx

```diff
@@ -7,6 +7,7 @@ import ErrorAlert from '../common/ErrorAlert';
 import { Employee, Project, Timesheet } from '@/lib/types/apiTypes';
 import { CreateTimesheetForm } from '../form/TimesheetForm';
 import { useUpdateTimesheets } from '@/lib/hooks/useTimesheets';
+import router from 'next/router';
```

Comment:
It's probably better to use `useRouter` for consistency (see e.g. this https://github.com/three-consulting/epoc-frontend/blob/main/pages/projects/index.tsx#L13).